imagersite/imagersite/urls.py

```diff
@@ -14,9 +14,22 @@
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
 from django.conf.urls import include, url
+from django.conf import settings
 from django.contrib import admin
+from .views import HomeView
+from django.conf.urls.static import static
+
 
 urlpatterns = [
-    url(r'^imager_profile/', include('imager_profile.urls')),
     url(r'^admin/', admin.site.urls),
+    url(r'^images/', include('imager_images.urls')),
+    url(r'^profile/', include('imager_profile.urls')),
+    url(r'^$', HomeView.as_view(), name='home_page'),
+    url(r'^images/', include('imager_images.urls')),
```

Comment:
The `images/` route is defined twice, in exactly the same way.

There is no route defined for `library/`, which is causing some issues.